app/controllers/confirmation-page.ts

```diff
@@ -0,0 +1,33 @@
+import { NextFunction, Request, Response, Router } from 'express';
+import moment from 'moment';
+import { paths } from '../paths';
+
+function getConfirmationPage(req: Request, res: Response, next: NextFunction) {
+  try {
+    const date = moment().add(14,'days').format('Do MMMM YYYY');
+    res.render('confirmation-page.njk', { date: date });
+  } catch (e) {
+    next(e);
+  }
+}
+
+function postConfirmationPage(req: Request, res: Response, next: NextFunction) {
```

Comment:
We dont need to send/save any data, so instead of a form add href to the button in the view